make/stratos/run

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+. "${BASH_SOURCE%/*}/_env"
+. "${BASH_SOURCE%/*}/_has_release.sh"
```

Comment:
Looks like the other helpers are in `make/include/*`.

It might make sense to call this `make/stratos-run` (to match `make/uaa-run`), and have the helper be `make/include/has-release` or something.